Gopkg.lock

```diff
@@ -98,12 +98,12 @@
   version = "v0.5.2"
 
 [[projects]]
-  digest = "1:df4af3a8e15cd72b068b8d898598001935edce4eac4eb5a3d7fd906036123ff4"
+  digest = "1:2debc00eaf8ffdbfb0570a791a84e64a3cfc146f5e61b54d4abd65c2de364552"
   name = "github.com/coreos/go-iptables"
   packages = ["iptables"]
   pruneopts = "UT"
-  revision = "259c8e6a4275d497442c721fa52204d7a58bde8b"
-  version = "v0.2.0"
+  revision = "f901d6c2a4f2a4df092b98c33366dfba1f93d7a0"
+  version = "v0.4.5"
```

Comment:
you're setting a constraint to v0.4.1 but this updates it to 0.4.5?

Reply:
Huh? I just run dep I have no idea how it happened, will fix.

Reply:
Oh I see... actually the constraint in dep reads like this `^0.4.1` so 0.4.5 is right